tests/test_app.py

```diff
@@ -1,14 +1,28 @@
+import mock
 import unittest
+
 from ubuntucleaner.main import UbuntuCleanerWindow
 
 
-class TestApp(unittest.TestCase):
+class TestUbuntuCleanerWindow(unittest.TestCase):
     def setUp(self):
         self.window = UbuntuCleanerWindow()
 
+    def tearDown(self):
+        del self.window
+
     def test_app(self):
         self.assertEqual(self.window.feature_dict, {'janitor': 0})
         self.assertIsNotNone(self.window.aboutdialog)
 
-    def tearDown(self):
-        del self.window
+    def test_on_about_button_clicked(self):
+        with mock.patch("ubuntucleaner.main.Gtk.AboutDialog.run") as m_run:
+            self.window.about_button.clicked()
+        self.assertEqual(m_run.call_args_list, [mock.call()])
+
+    def test_on_mainwindow_destroy(self):
+        with mock.patch("ubuntucleaner.main.Gtk.main_quit") as m_main_quit, \
+                mock.patch("ubuntucleaner.main.exit") as m_exit:
```

Comment:
Mocks both so it doesn't crashes on-going unit tests with an `SIGINT`. Also helps in verifying `on_mainwindow_destroy()` got called as expected